Updated for new prod, added local run option

beyerja opened the pull request:
BEGINRELEASENOTES
- Updated bash scripts, now also possible to run on already reconstructed files (DSTs).
- Added vector boson mass lines to plots. 
- Added another FastJet processor to the Marlin script to remove overlay which is included in new production.

ENDRELEASENOTES

Commit:
Updated for now prod, added local run option

Updated bash scripts, now also possible to run on already reconstructed
files (DSTs). Added vector boson mass lines to plots. Added another
FastJet processor to the Marlin script to remove overlay which is
included in new production.

rete reviewed (changes requested):
The content itself looks nice, but you need to fix the indentation

rete commented on WWZZSeparation/plotting/create_mjjmjjplots.cc:
```diff
@@ -109,18 +112,47 @@ void create_mjjmjjplots(
+  float h2_y_min = gPad->GetFrame()->GetY1();
+  float h2_y_max = gPad->GetFrame()->GetY2();
+
+	TLine* h2_W_xline = new TLine(h2_x_min, M_W, h2_x_max, M_W);
```
Memory leaaaaaaaaak !

Commit:
Fixing indentation, avoid memory leak

Switched indentation in all C++ files and some others to spaces. Added
deletes to avoid memory leak in ROOT macro.

rete commented:
Looks fine now ! Danke !
